browser/ui/views/tabs/brave_compound_tab_container.cc

```diff
@@ -529,16 +528,19 @@ void BraveCompoundTabContainer::ScrollTabToBeVisible(int model_index) {
   // Unfortunately, ScrollView's API doesn't work well for us. So we manually
   // adjust scroll offset. Note that we change contents view's position as
   // we disabled layered scroll view.
+
+  // Scroll UP
   if (visible_rect.CenterPoint().y() >=
       tab_bounds_in_contents_view.CenterPoint().y()) {
-    scroll_view_->contents()->SetPosition(
-        {0, -static_cast<int>(tab_bounds_in_contents_view.y())});
-  } else {
-    scroll_view_->contents()->SetPosition(
-        {0, std::min(0, scroll_view_->height() -
+    scroll_view_->ScrollToOffset(
```

Comment:
nit: I think having comment in the block would be better
```suggestion
    // Scroll up
    scroll_view_->ScrollToOffset(
```